### PR TITLE
Minor doc updates

### PR DESCRIPTION
**Pull request by allenleetc**
## What changes are proposed in this pull request?

Minor doc/tutorial updates after testing in fresh environments

## How is this patch tested? If it is not, please explain why.

Manual testing of notebooks on Linux, reading html

### Is this a user-facing change that should be mentioned in the release notes?

-   [x] No. You can skip the rest of this section.
-   [ ] Yes. Give a description of this change to be included in the release
        notes for FiftyOne users.

(Details in 1-2 sentences. You can just refer to another PR with a description
if this PR is part of a larger change.)

### What areas of FiftyOne does this PR affect?

-   [ ] App: FiftyOne application changes
-   [ ] Build: Build and test infrastructure changes
-   [ ] Core: Core `fiftyone` Python library changes
-   [x] Documentation: FiftyOne documentation changes
-   [ ] Other

### CODE REVIEW

File: docs/source/faq/index.rst [hunk 1]
```diff
@@ -158,7 +158,7 @@ including the following commands in your notebook before creating any plots:
     import plotly.offline as po
     po.init_notebook_mode(connected=True)
 
-Note that FiftyOne currently requires `plotly>=4.14,<5`, which should have been
+Note that FiftyOne currently requires `plotly`, which should have been
```
Comment (brimoor) [inline docs/source/faq/index.rst:161]: Since we no longer have an upper bound requirement on `plotly`, perhaps this paragraph can just be deleted?

File: fiftyone/zoo/models/manifest-torch.json [hunk 2]
```diff
@@ -501,7 +501,7 @@
             "base_name": "keypoint-rcnn-resnet50-fpn-coco-torch",
             "base_filename": "keypointrcnn_resnet50_fpn_coco-fc266e95.pth",
             "version": null,
-            "description": "Keypoint R-CNN model from `Keypoint Density-based Region Proposal for Fine-Grained Object Detection and Classification using Regions with Convolutional Neural Network Features <https://arxiv.org/abs/1603.00502>`_ with ResNet-50 FPN backbone trained on COCO",
+            "description": "Keypoint R-CNN model from `Mask R-CNN <https://arxiv.org/abs/1703.06870>`_ with ResNet-50 FPN backbone trained on COCO",
```
Comment (brimoor) [inline fiftyone/zoo/models/manifest-torch.json:504]: Just confirming: looks like we were linking to the wrong paper here?

Comment (allenleetc) [reply fiftyone/zoo/models/manifest-torch.json:504]: Yes believe so, for instance `keypoint-rcnn-resnet50-fpn-coco-torch` on the Pytorch models page also links to Mask R-CNN. It's a bit confusing given the titles but the previously-linked paper doesn't actually predict keypoints (it uses keypts internally for ROI selection to do objdet). Will double-check

File: docs/source/faq/index.rst (hunk 1 above)
Comment (allenleetc) [reply docs/source/faq/index.rst:161]: Sounds good will update

File: fiftyone/zoo/models/manifest-torch.json (hunk 2 above)
Comment (allenleetc) [reply fiftyone/zoo/models/manifest-torch.json:504]: confirmed previous link was incorrect
